### PR TITLE
[edit] Edits to MFA pages, replaced screenshot and renamed to lowercase

### PR DESCRIPTION
**Pull request by Catografix**
The '/multifactor-authentication/custom-provider.md' file contains no information not already in 'index.md' and should be deleted. I dont see it linked to elsewhere. I can add a commit to delete this file to this PR if desired.
Also see line comment below.

### CODE REVIEW

File: articles/multifactor-authentication/yubikey.md [hunk 1]
```diff
@@ -194,47 +200,41 @@ return function (context, req, res) {
...
 
 Once the Webtask CLI is installed, run:
 
-```
+```txt
 wt create --name yubikey-mfa --secret yubikey_secret={YOUR YUBIKEY SECRET} --secret yubikey_clientid={YOUR YUBIKEY CLIENT ID} --secret returnUrl=https://${account.namespace}/continue --output url --profile {WEBTASK PROFILE} yubico-mfa-wt.js
 ```
 
-The `WEBTASK PROFILE` is the value obtained from the __Account Settings__ above.
+The `WEBTASK PROFILE` is the value obtained from the Account Settings page.
```
Comment (Catografix) [inline articles/multifactor-authentication/yubikey.md]: I dont see where the WEBTASK PROFILE value is indicated in the Account Settings page.

Comment (ntotten) [inline articles/multifactor-authentication/yubikey.md]: Yeah, this should be more clear. In step 2 in the Account Settings -> Web Task page, it is the value of the `-p` parameter in the command.

**Comment by ntotten** [comment]
@Catografix Please delete. Also, add a redirect in this file: https://github.com/auth0/docs/blob/master/redirect-urls.json

Just redirect the old page to the index page or whatever makes sense.

**Comment by Catografix** [comment]
Made changes, Ready to go.

File: redirect-urls.json [hunk 2]
```diff
@@ -555,4 +555,8 @@
     "from": "/premium-support",
     "to": "/support"
   }
+  {
+    "from": "/multifactor-authentication/custom-provider.md",
```
Comment (ntotten) [inline redirect-urls.json]: no `.md` extensions here. just the url, not the doc path.

**Comment by Catografix** [comment]
fixed, thanks

**Pull request merged**